### PR TITLE
style:: disable buttons now look like disable buttons

### PR DESCRIPTION
**Pull request by Matasin**

### CODE REVIEW

File: frontend/src/assets/stylesheets/components/post-buttons.sass [hunk 1]
```diff
@@ -21,6 +21,8 @@
     text-transform: uppercase
     width: 180px
 
+    transition: $transition-duration $transition-function
```
Comment (LukaszBielsko) [inline frontend/src/assets/stylesheets/components/post-buttons.sass:24]: looks nice now :)

File: frontend/src/assets/stylesheets/components/post-buttons.sass [hunk 2]
```diff
@@ -21,6 +21,8 @@
...
+
     &:disabled
```
Comment (LukaszBielsko) [inline frontend/src/assets/stylesheets/components/post-buttons.sass:26]: Cursor pointer when hovered over disabled button (hand with pointing finger) implies that this button might actually be clickable. Please change to something more suitable.

File: frontend/src/assets/stylesheets/components/post-buttons.sass [hunk 3]
```diff
@@ -46,10 +49,10 @@
     width: 180px
 
   &__button-logout
-    background: none;
```
Comment (LukaszBielsko) [inline frontend/src/assets/stylesheets/components/post-buttons.sass:49]: 🤣

File: frontend/src/assets/stylesheets/components/post-buttons.sass (hunk 1 above)
Comment (Matasin) [reply frontend/src/assets/stylesheets/components/post-buttons.sass:24]: :)

File: frontend/src/assets/stylesheets/components/post-buttons.sass (hunk 2 above)
Comment (Matasin) [reply frontend/src/assets/stylesheets/components/post-buttons.sass:26]: Good point, there should be cursor not allowed from begining

File: frontend/src/assets/stylesheets/components/post-buttons.sass (hunk 3 above)
Comment (Matasin) [reply frontend/src/assets/stylesheets/components/post-buttons.sass:49]: Weird i do not have semicolon there

Screenshot 2021-06-17 at 13 49 05

Comment (Matasin) [reply frontend/src/assets/stylesheets/components/post-buttons.sass:49]: hmm, is about semicolon?

**Pull request merged**